### PR TITLE
now items are shown one time per iteration

### PR DESCRIPTION
**Pull request by danilofreitas**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -9,33 +10,47 @@
...
-  };
-
-  $(document).keyup(function(k) {
-    if (k.keyCode == 32) {
-      $("h1").fadeTo(getRandomExcuse());
+    ];
+
+    var excusesHolder = [];
+
+    function getRandomExcuse() {
```
Comment (onlab) [inline main.js:17]: What about
if (randomExcuses.length == 0) {
  randomExcuses = _ shuffle(excusesHolder);
}

var currentExcuse = randomExcuses.pop();
excusesHolder.push(currentExcuse);

return currentExcuse;

File: main.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-(function($) {
+(function($, _) {
 
-  var randomExcuses = [
+  $(window).load(function () {
+    var randomExcuses = [
```
Comment (onlab) [inline main.js:4]: You can _ shuffle([....]) here

Comment (danilofreitas) [inline main.js:4]: Sure, and it's there in the first time it's there, but I would to keed the previous logic

File: main.js (hunk 1 above)
Comment (danilofreitas) [inline main.js:17]: Yes, I can do it, looks better, but I'll add the reset to the excusesHolder

**Comment by ferodss** [comment]
Seems like is not working very well. @danilofreitas promised me a better implementation

**Pull request closed without merging**